src/gc/gcpriv.h

```diff
@@ -3002,6 +3002,17 @@ class gc_heap
     PER_HEAP_ISOLATED
     size_t min_segment_size;
 
+    // For SOH we always allocate segments of the same
+    // size unless no gc region requires larger ones.
```

Comment:
the actual implementation is all named something no_gc_region, that's why I referred to it as no gc region. I guess I can change it to no_gc_region if that seems better.